sky/sdk/example/address_book/lib/main.dart

```diff
@@ -81,24 +100,25 @@ class AddressBookApp extends App {
   }
 
   Widget build() {
-
     ThemeData theme = new ThemeData(
       brightness: ThemeBrightness.light,
       primarySwatch: colors.Teal,
       accentColor: colors.PinkAccent[100]
     );
-
     return new Theme(
       data: theme,
-        child: new DefaultTextStyle(
-          style: typography.error, // if you see this, you've forgotten to correctly configure the text style!
-          child: new TaskDescription(
-            label: 'Address Book',
+      child: new DefaultTextStyle(
+        style: typography.error, // if you see this, you've forgotten to correctly configure the text style!
+        child: new TaskDescription(
+          label: 'Address Book',
+          child: new Focus(
+            defaultFocus: nameKey,
```

Comment:
Who else would do it?

I'm open to other options. I think it makes sense, though. You're describing the state of the app, and one aspect about the app that you're describing is what the default focus should be. Consider a dialog box that you've just popped up. You'd need to specify the default there, too, and you'd need to specify it the first time the dialog becomes relevant each time.

I don't really see another way to do it.